lib/config/presets/internal/group.ts

```diff
@@ -111,6 +111,15 @@ const staticGroups = {
       },
     ],
   },
+  drupalCore: {
+    description: 'Group Drupal core updates.',
+    packageRules: [
+      {
+        groupName: 'Drupal core',
+        matchPackageNames: ['drupal/core', '/^drupal/core-.*/'],
```

Comment:
```suggestion
        matchPackageNames: ['drupal/core', 'drupal/core-*'],
```

I think this is the same and simpler?